### DISCUSSION TITLE
lvmd on ARM support level

### DISCUSSION THREAD

**Discussion by matthias50** [topolvm/topolvm#672] [Q&A] [answered]
I noticed that there isn't an lvmd binary in the releases. Is this intentional or an oversight? On a related note, is the daemonset based deployment of lvmd officially supported or is this still experimental? And is the lvmd daemonset deployment equally supported on x64 and arm?

**Comment by peng225** [comment, marked as answer]
@matthias50 
> I noticed that there isn't an lvmd binary in the releases. Is this intentional or an oversight?

Sorry, but this is oversight.

> On a related note, is the daemonset based deployment of lvmd officially supported or is this still experimental?

The daemonset-based deployment of lvmd is now supported. That is the current default.
https://github.com/topolvm/topolvm/tree/main/charts/topolvm#quick-start

> And is the lvmd daemonset deployment equally supported on x64 and arm?

Though we are building container for both x64 and arm, it is not tested in the arm environment.
